include/bluetooth/conn.h

```diff
@@ -279,15 +279,22 @@ struct bt_conn *bt_conn_create_slave_le(const bt_addr_le_t *peer,
 /** Security level. */
 typedef enum __packed {
 	/** Only for BR/EDR special cases, like SDP */
-	BT_SECURITY_NONE,
+	BT_SECURITY_L0,
 	/** No encryption and no authentication. */
```

Comment:
Can you add: `Level 1: No encryption and no authentication`
same with the rest

Reply:
Done.